database/migrations/2024_11_29_125731_add_columns_to_discounts_table.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration
+{
+    public function up(): void
+    {
+        Schema::table('discounts', function (Blueprint $table) {
+            $table->string('name')->nullable()->after('model_id');
+            $table->decimal('discount_percentage', 40, 10)->nullable()->after('discount');
+            $table->decimal('discount_currency', 40, 10)->nullable()->after('discount_percentage');
```

Comment:
I don't get why this column is named that way. To me this implies the currency of the discount, which does not make any sense to me.

Reply:
do you have a better suggestion? the discount_currency displays the discount in € or $ or whatever the order is.